libafl/src/events/llmp/mod.rs

```diff
@@ -321,6 +321,13 @@ where
                 }
                 Ok(())
             }
+
+            #[cfg(feature = "share_objectives")]
```

Comment:
here you should save the input as an objective.
you should look at fuzzer.rs's `fn process_execution` to see how it is done

Reply:
I suppose it is acceptable to get the input (to be saved) from the given `State`, or should I get the input in some other way?

Reply:
the input should be included in Event::Objective. so you pass it over the wire.
you should change its definition in Event::Objective in events/mod.rs